lib/pages/chat/chat_pinned_events/pinned_messages.dart

```diff
@@ -219,6 +227,113 @@ class PinnedMessagesController extends State<PinnedMessages>
     );
   }
 
+  List<ContextMenuItemChatAction> listHorizontalActionMenuBuilder() {
```

Comment:
I see different styles of context menu in the same screen. Why not use the same kind of context menu?

Reply:
We use the same style as messages list, we don't display the same button in the context menu, I had to clone all the context, hover logic because it was exclusive to chat list

Reply:
Done, Context menu is the same as messages list one